src/systemd/cockpit-ws-user.service

```diff
@@ -1,14 +0,0 @@
-[Unit]
```

Comment:
This is also still referenced here:

test/verify/check-connection:        m.execute("systemctl stop cockpit-ws-user.service")
tools/debian/cockpit-ws.install:${env:deb_systemdsystemunitdir}/cockpit-ws-user.service

which ought to be simple to fix. But the more interesting place is in tools/debian/tests/smoke which needs heavier adjustment. Since this is very difficult to set up (it heavily depends on how the ci.debian.net production setup currently looks like), I think we should just drop the hack for now, I'll do the next cockpit upload to Debian, see it fail, and then sort it out downstream first.

Reply:
I indeed dropped this locally.

Reply:
Done.